lib/fastlane/plugin/wpmreleasetoolkit/helper/git_helper.rb

```diff
@@ -95,6 +95,15 @@ def self.commit(message:, files: nil, push: false)
         end
       end
 
+      # Get the sha1 of a given git ref. Typically useful to get the sha1 of the current HEAD commit.
+      #
+      # @param [String] ref The git ref (commit, branch name, 'HEAD', …) to resolve as a sha1
+      # @return [String] The commit sha1 of the ref
+      #
+      def self.get_commit_sha(ref: 'HEAD')
```

Comment:
This may be overly nitpicky, but [the release toolkit already depends on the `git` gem](https://github.com/wordpress-mobile/release-toolkit/blob/c476c50f507093ed30070740803083a4d4a42b49/fastlane-plugin-wpmreleasetoolkit.gemspec#L37) – could we use that instead of adding our own helper?

Reply:
Yeah good point will look into it.

Reply:
Fixed in 2809f2f5d9492c1bb5332d05b533aaba04cbba25